safe_extras/parameters/qt_widgets/test/test_boolean_parameter_widget.py

```diff
@@ -1,53 +0,0 @@
-# coding=utf-8
```

Comment:
The rationale is we take safe_extras as outside code, we don't need to test them as they are not our code. And it doesn't effect to our test coverage since we only cover safe packager.

But, for parameters, it's actually not inasafe's code, it's from different repo (https://github.com/AIFDR/parameters). Although the developers are in the same team.